kubernetes/onpremise/kubeadm/scripts/install_kubernetes.sh

```diff
@@ -0,0 +1,177 @@
+# install Kubectl
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+curl -LO "https://dl.k8s.io/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"
+echo "$(<kubectl.sha256) kubectl" | sha256sum --check
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+# install Kubernetes
+modprobe br_netfilter
+cat <<EOF | sudo tee /etc/modules-load.d/k8s.conf
+br_netfilter
+EOF
+
+cat <<EOF | sudo tee /etc/sysctl.d/k8s.conf
+net.bridge.bridge-nf-call-ip6tables = 1
+net.bridge.bridge-nf-call-iptables = 1
+EOF
+sudo sysctl --system
+
+cat <<EOF | sudo tee /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch
+enabled=1
+gpgcheck=1
+repo_gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+exclude=kubelet kubeadm kubectl
+EOF
+
+sudo setenforce 0
+sudo sed -i 's/^SELINUX=enforcing$/SELINUX=permissive/' /etc/selinux/config
+sudo yum install -y kubelet kubeadm kubectl --disableexcludes=kubernetes
+sudo systemctl enable --now kubelet
+
+sudo yum install -y kubelet kubeadm
+sudo systemctl enable kubelet && systemctl start kubelet
+
+mkdir -p /etc/docker
+cat <<EOF | sudo tee /etc/docker/daemon.json
+{
+"exec-opts": ["native.cgroupdriver=systemd"],
+"log-driver": "json-file",
+"log-opts": {
+    "max-size": "100m"
+},
+"storage-driver": "overlay2"
+}
+EOF
+
+sudo systemctl enable docker
+sudo systemctl daemon-reload
+sudo systemctl restart docker
+
+sudo mkdir -p /etc/containerd
+sudo touch /etc/containerd/config.toml
+sudo chown ${user}:${user} /etc/containerd/config.toml
+sudo containerd config default > /etc/containerd/config.toml
+sudo sed -i '/sandbox_image/s#registry.k8s.io/pause:3.6#registry.k8s.io/pause:3.9#' /etc/containerd/config.toml
+sudo chown root:root /etc/containerd/config.toml
+cat /etc/containerd/config.toml | grep -i sandbox_image
+sudo systemctl restart containerd
+
+
+%{ if node == "master" ~}
```

Comment:
I am not sure it is a good idea to handle this via terraform. It might be better to use environment variables, and do those tests directly in shell script.